examples/tcp-socket-listener-client/tcp_socket_client.bal

```diff
@@ -0,0 +1,55 @@
+// This is the client implementation for the TCP socket with the attached callback service. Callback service is optional.
+import ballerina/io;
+import ballerina/socket;
+
+public function main() {
+    // Create new socket client by providing host, port and callback service.
+    socket:Client socketClient = new({ host: "localhost", port: 61598,
+            callbackService: ClientService });
+    string content = "Hello Ballerina";
+    byte[] payloadByte = content.toByteArray("UTF-8");
+    // Send desire content to the server using write function.
+    var writeResult = socketClient->write(payloadByte);
+    if (writeResult is error) {
+        io:println("Unable to written the content ", writeResult);
+    }
+}
+
+// Callback service for the TCP client. Service need to have below four predefine resources.
+service ClientService = service {
+
+    // This invokes once the client connect to the TCP server.
```

Comment:
```suggestion
    // This gets invoked once the client connects to the TCP server.
```
or ` This is invoked..` . Shall we fix in all places?

Reply:
@samgnaniah Can you please share your thought on here?